apps/arena/lib/arena/game_updater.ex

```diff
@@ -33,10 +33,10 @@ defmodule Arena.GameUpdater do
   # END API
   ##########################
 
-  def init(%{clients: clients}) do
-    game_id = self() |> :erlang.term_to_binary() |> Base58.encode()
+  def init(%{clients: clients, missing_clients: missing_clients}) do
```

Comment:
Wait! The game shouldn't know that some clients are bots. This logic should live inside the launcher.
Regarding the core of the game, all the clients are equal.

Please move it to the `game_launcher` file.

Reply:
you're right! the game should only know about the game state and how to change it, moved the bot initialization to the game launcher as suggested, i also updated the links with the bots names and updated the mermaid diagrams